myuw/dao/instructor_schedule.py

```diff
@@ -163,3 +163,23 @@ def is_instructor(request):
             return False
 
         raise
+
+
+def is_section_instructor(section_label):
+    """
+    Determines if user is an instructor of the given section label
+    """
+    try:
+        person = get_person_of_current_user()
+        print "per"
+        section = get_section_by_label(section_label)
+        print section_label
+        print section.is_instructor(person)
+        return section.is_instructor(person)
```

Comment:
In the case of email list, if the section is secondary, you may check if the user is instructor of its primary section.